software/crypt.fyi.yaml

```diff
@@ -0,0 +1,14 @@
+name: crypt.fyi
+website_url: https://crypt.fyi
+description: A zero-knowledge, end-to-end encrypted secret sharing platform that enables secure transmission of sensitive information. Features include AES-256-GCM encryption, burn after reading, automatic expiration, password protection, file sharing, webhook notifications, and IP/CIDR allow-listing.
+licenses:
+  - Apache-2.0
+platforms:
+  - Javascript
+  - Nodejs
+  - Docker
+tags:
+  - Communication - Custom Communication Systems
```

Comment:
This tag is out the scope, its more for messaging systems and stuff like that.
```suggestion
```

Reply:
Ah - I was following yopass's tagging which has a similar core feature-set to crypt.fyi (encrypt a message and send the secure one-time link to recipient) https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/9dd3f5e038c94156f4a5893a9b2625fb56fff3f7/software/yopass.yml#L10

Reply:
I think @Ki-er is correct, and yopass's tags are wrong. Updated here https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/1108/files, please use the same tags